documentation/asciidoc/topics/proc_exchanging_bound_service_account_tokens.adoc

```diff
@@ -0,0 +1,65 @@
+[id="exchanging-bound-service-account-tokens-{context}"]
+= Exchanging bound service account tokens
+
+[role="_abstract"]
+Create service account tokens with a limited lifespan and add them into secrets at each backup location.
+You must refresh the token periodically to prevent {ispn_operator} from losing access to the remote {openshiftshort} cluster.
+For non-expiring tokens, see link:#exchanging-service-account-tokens-cross-site[Exchanging service account tokens].
+
+.Prerequisites
+
+* You have created a service account.
+
+.Procedure
+
+. Log in to your {openshiftshort} cluster.
+
+. Create a bound token for the service account:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> create token <service-account>
+----
++
+
+[NOTE]
+====
+By default, service account tokens are valid for one hour.
+Use the command option `--duration` to specify the lifespan in seconds..
+====
++
+The command prints the token in the terminal.
+
+. Copy the token for deployment in the backup {openshiftshort} cluster(s).
+
+. Log in to the backup {openshiftshort} cluster.
+
+. Add the service account token for a backup location:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> create secret generic <token-secret> --from-literal=token=<token>
+----
++
+The `<token-secret>` is the name of the secret configured in the `Infinispan` CR.
+
+. Repeat the steps on other {openshiftshort} clusters.
+
+[discrete]
+== Deleting expired tokens
+When a token expires, delete the expired token, and then repeat the procedure to generate and exchange a new one.
+
+. Log in to the backup {openshiftshort} cluster.
+
+. Delete the expired secret `<token-secret>`:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+oc -n <namespace> delete secrets <token-secret>
+----
+
+. Repeat the procedure to create a new token and generate a new `<token-secret>`.
+
+[role="_additional-resources"]
+.Additional resources
+* link:{link_os_sa_tokens}
```

Comment:
should link to bound service tokens. should I fix it? @domiborges

Reply:
Correct. Now it's updated